packages/wonder-blocks-button/src/components/button.md

```diff
@@ -420,12 +420,47 @@ const styles = StyleSheet.create({
     </View>
 </MemoryRouter>
 ```
+### Running callbacks on navigation
```

Comment:
This is really great! and totally fine to leave it here until we finish the docs migration.

Could you please replicate this section in Storybook?

Probably this could be an option: https://github.com/Khan/wonder-blocks/blob/master/packages/wonder-blocks-button/src/components/__docs__/best-practices.stories.mdx

But if you think this deservers to be on its own section, then we can create a new `.mdx` file in that folder to show this.